src/components/3d-hover-effect-glow/3d-hover-effect-glow.scss

```diff
@@ -0,0 +1,21 @@
+:host(limel-3d-hover-effect-glow) {
+    transition: background, opacity;
+    transition-timing-function: ease-out; // Must be the same as the 3D Element itself
+    transition-duration: 0.8s; // Must be the same as the 3D Element itself
+
+    display: block;
+    pointer-events: none;
+
+    position: absolute;
+    inset: 0;
+
+    opacity: 0.1;
```

Comment:
The problem with putting the styling here is that the component will be rendered before the styling applies. It won't be visible then, since it's unstyled, but it will have a default opacity of 1. Then the styling applies, which set's the opacity to 0.1. But there's also been a transition applied to the opacity, so there will be this little "lightning flash" when the component is loaded.

Reply:
https://github.com/user-attachments/assets/dfc087da-b7d2-411a-b7f0-bea5fd338fc7

Reply:
😄 On my slow machine things load in a totally different way and jump around so much until the page is completely loaded, so that I can't even see this visual effect that you screen recorded.

Bu I think I can fix it. I'll adda. fixup

Reply:
### A note for future:
There are several ways of mitigating the effect that you mentioned.

The best one would be using [the `@property` rule](https://developer.mozilla.org/en-US/docs/Web/CSS/@property) to animate the `x` and `y`, and opacity values of the radial gradient. I tried it, but couldn't really make it work. I don't exactly know why. Due to lack of time, I decided to keep a copy of the code here, and do it in a simpler but dirtier way. 

What I did was like below…
In `3d-hover-effect-glow.scss` tried to `transition` the CSS custom properties, combining them with `@property` rule.

```scss
:host(limel-3d-hover-effect-glow) {
    @property --limel-3d-hover-effect-glow-opacity {
        syntax: '<number>';
        inherits: true;
        initial-value: 0.1;
    }

    @property --limel-3d-hover-effect-glow-position-x {
        syntax: '<percentage>';
        inherits: true;
        initial-value: 50%;
    }

    @property --limel-3d-hover-effect-glow-position-y {
        syntax: '<percentage>';
        inherits: true;
        initial-value: -20%;
    }

    transition-property: --limel-3d-hover-effect-glow-opacity,
        --limel-3d-hover-effect-glow-position-x,
        --limel-3d-hover-effect-glow-position-y;
    transition-timing-function: ease-out; // Must be the same as the 3D Element itself
    transition-duration: 0.8s; // Must be the same as the 3D Element itself

    display: block;
    pointer-events: none;

    position: absolute;
    inset: 0;

    background-image: radial-gradient(
        circle at var(--limel-3d-hover-effect-glow-position-x, 50%)
            var(--limel-3d-hover-effect-glow-position-y, -20%),
        rgb(var(--color-white), var(--limel-3d-hover-effect-glow-opacity, 0.1)),
        rgb(var(--color-white), 0)
    );

    mix-blend-mode: plus-lighter;
}
```

and then in the `mixins.scss`, I would do:
```scss

@mixin the-3d-element {
    
   // …

    &:hover {
        limel-3d-hover-effect-glow {
            --limel-3d-hover-effect-glow-opacity: 0.5;
            @media (prefers-reduced-motion) {
                --limel-3d-hover-effect-glow-opacity: 0.2;
            }
        }
    }
}
```

and of course the JS in `3d-tilt-hover-effect.ts` was updated too:

```tsx
        const glowPositionX = `
    ${center.x * GLOW_POSITION_MULTIPLIER + the3dElementBounds.width / CENTER_DIVISOR}px
`;
        const glowPositionY = `
    ${center.y * GLOW_POSITION_MULTIPLIER + the3dElementBounds.height / CENTER_DIVISOR}px
`;
        element.style.setProperty(
            '--limel-3d-hover-effect-glow-position-x',
            glowPositionX,
        );
        element.style.setProperty(
            '--limel-3d-hover-effect-glow-position-y',
            glowPositionY,
        );
    };
```
and
```tsx
    const handleMouseLeave = () => {
        document.removeEventListener('mousemove', tiltCallback);
        element.style.removeProperty('--limel-3d-hover-effect-rotate3d');
        element.style.removeProperty('--limel-3d-hover-effect-glow-position-x');
        element.style.removeProperty('--limel-3d-hover-effect-glow-position-y');
    };

```

Reply:
Yeah, I've noticed there's a bit of a jump too.

But, can we put this PR on ice until cooldown? (Or if you can convince someone appropriate that I should spend cycle time on this, then I'm happy to help out and review further 😅)

Reply:
Yes, all is done on this PR for now. And this can go to cooldown.

But now I don't know which cooldown board 😄, since we have now several cooldown boards 🙈